config.js

```diff
@@ -18,14 +18,14 @@ module.exports = {
   taskflowDB: {
     database: process.env.DATABASE_NAME || 'taskflow',
     host: process.env.DATABASE_HOST,
-    password: process.env.DATABASE_PASSWORD,
+    password: process.env.DATABASE_PASSWORD || 'test-password',
     port: process.env.DATABASE_PORT,
     user: process.env.DATABASE_USERNAME || 'postgres'
   },
   db: {
     database: process.env.ASL_DATABASE_NAME || 'asl',
     host: process.env.ASL_DATABASE_HOST,
-    password: process.env.ASL_DATABASE_PASSWORD,
+    password: process.env.ASL_DATABASE_PASSWORD || 'test-password',
```

Comment:
Does anything _not_ work if these are not specified?

Reply:
`npm run migrate` seems to rely on them being in `config.js` and `npm run seed` needs them in `knexfile.js`

Reply:
both seem to ignore the `.env`